css/mobile_first.css

```diff
@@ -46,10 +46,10 @@ body {
 
 .external_link a {
   text-decoration: none;
-  font-weight: 50;
+  font-weight: 500;
   font-size: 15px;
   line-height: 20px;
-  color: #fff;
+  color: #595959;
   text-align: center;
   resize: none;
 }
```

Comment:
Fixed _accessibility issue_ 2.
> As requiered from the last review:
 - Changed `font-weight` of some elements¹ that had **Contrast Errors**.
 - Changed `color` of the same elements¹.